### PR TITLE
[Doc/Build/Testing] rename fuse-query to datafuse

### PR DESCRIPTION
**Pull request by ZhiHanZ**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary

Currently, we have already build release in multi-arch platform(linux/arm64, linux/amd64, linux/arm/v7, linux/arm/v6) and support to install fuse-query and fuse-store in one install script(https://github.com/datafuselabs/datafuse/pull/1121), thus far we can migrate from original datafusedev to datafuselabs docker host with those platform supports.

We should check the following part integrate well with this update
1. website installation page should update
2. Makefile should change default docker host to datafuselabs
3. CI part should change secret.DOCKERHUB_NAMESPACE conform to the renaming

## Changelog

- Build/Testing/CI
- Documentation
- Website
- Other 
- Not for changelog (changelog entry is not required)

## Related Issues

fixes https://github.com/datafuselabs/datafuse/issues/1132
fixes https://github.com/datafuselabs/datafuse/issues/1048
fixes https://github.com/datafuselabs/datafuse/issues/893
Should also
fixes https://github.com/datafuselabs/datafuse/issues/1109
## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-HUB ?= datafusedev
+HUB ?= datafuselabs
```
Comment (BohuTANG) [inline Makefile:1]: :+1:

**Review by BohuTANG — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (BohuTANG) [inline Makefile:1]: Need add datafuselabs key to env?

**Comment by PsiACE** [comment]
@ZhiHanZ  see also: https://github.com/datafuselabs/datafuse/blob/master/scripts/deploy/fusequery-standalone-from-release.sh

**Pull request merged**